pom.xml

```diff
@@ -849,6 +849,7 @@
     <commons-validator.version>1.7</commons-validator.version>
     <commons-io.version>2.14.0</commons-io.version>
     <commons-lang3.version>3.9</commons-lang3.version>
+    <commons-text.version>1.10.0</commons-text.version>
```

Comment:
This is actually not the latest version (latest is 1.13.0) but this is the version which we HBase had transitively via `org.apache.hadoop:hadoop-common:jar:3.4.1`.